src/mpybit.py

```diff
@@ -0,0 +1,107 @@
+                                        (
+                                            'createRandomAddress', 4,
+                                            streamNumberForAddress, label, 1,
+                                            "", eighteenByteRipe, nonceTrialsPerByte,
+                                            payloadLengthExtraBytes
+                                        )
+        )
+        print(BMConfigParser().sections(), "BMConfigParser().sections()")
+        fromAddress = queues.apiAddressGeneratorReturnQueue.get()
+        print("BREAK POINT ENDING")
+        toAddress = "BM-2cWyUfBdY2FbgyuCb7abFZ49JYxSzUhNFe"
+        message = self.ids.user_input.text
+        subject = 'Test'
+        encoding = 3
+        print("message: ", self.ids.user_input.text)
+        sendMessageToPeople = True
+        if sendMessageToPeople:
+            if toAddress != '':
+                status, addressVersionNumber, streamNumber, ripe = decodeAddress(
+                    toAddress)
+                if status == 'success':
+                    toAddress = addBMIfNotPresent(toAddress)
+
+                    if addressVersionNumber > 4 or addressVersionNumber <= 1:
+                        print("addressVersionNumber > 4 or addressVersionNumber <= 1")
+                    if streamNumber > 1 or streamNumber == 0:
+                        print("streamNumber > 1 or streamNumber == 0")
+                    if statusIconColor == 'red':
+                        print("shared.statusIconColor == 'red'")
+                    stealthLevel = BMConfigParser().safeGetInt(
+                        'bitmessagesettings', 'ackstealthlevel')
+                    ackdata = genAckPayload(streamNumber, stealthLevel)
+                    sqlExecute(
+                        '''INSERT INTO sent VALUES (?,?,?,?,?,?,?,?,?,?,?,?,?,?,?)''',
+                        '',
+                        toAddress,
+                        ripe,
+                        fromAddress,
+                        subject,
+                        message,
+                        ackdata,
+                        int(time.time()),
+                        int(time.time()),
+                        0,
+                        'msgqueued',
+                        0,
+                        'sent',
+                        encoding,
+                        BMConfigParser().getint('bitmessagesettings', 'ttl'))
+                    queues.workerQueue.put(('sendmessage', toAddress))
+                    print("sqlExecute successfully #####    ##################")
+                    for i in threading.enumerate():
+                        print(i.name)
+                    return None
+
+    def sayexit(self):
+        """This method will exit the application screen."""
+        print("**************************EXITING FROM APPLICATION*****************************")
+        shutdown.doCleanShutdown()
+        Window.close()
+
+
+class MainApp(App):
```

Comment:
Where it instantiated?

Reply:
already did it as we need to start all the necessary threads at the very starting so I have to explicity call bitmessage main inside this main module and it will than call MainApp().run()